### ISSUE TITLE
Search: Improve accessibility

### ISSUE THREAD

**Issue by ddprrt** [Dynatrace/css-groundhog#188]
The new search bar has empty links and no labels in the form. This usually is not good for A11y. Let's improve that by adding necessary texts or labels, or fall back to `aria-` attributes when necessary.
